src/main/resources/templates/db_scripts/schema.json

```diff
@@ -14,6 +14,13 @@
       "fromModuleVersion": 1.0,
       "withMetadata": true,
       "populateJsonWithId": true
+    },
+    {
+      "tableName": "vouchers",
```

Comment:
we probably want to create a unique index on the voucherNumber field, but we can handle that in https://issues.folio.org/browse/MODINVOSTO-12 (which I'll update in a moment)